media-sound/denemo/files/denemo-2.2.0-find-guile-2.2.patch

```diff
@@ -0,0 +1,96 @@
+diff --git a/configure.ac b/configure.ac
+index 4bb3afa..01394ef 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -30,7 +30,10 @@ AC_PROG_CXX
+ AC_PROG_INSTALL
+ AC_PROG_YACC
+ AM_PROG_LEX
++GUILE_PKG
+ GUILE_FLAGS
++CFLAGS="$CFLAGS $GUILE_CFLAGS"
++LIBS="$LIBS $GUILE_LIBS"
+ 
+ AC_PROG_LIBTOOL
+ 
+@@ -40,43 +43,43 @@ AS_COMPILER_FLAG([-fdiagnostics-color=auto],
+ 
+ AM_CONDITIONAL(ENABLE_GLIB_TEST, [test "x$enable_glibtest" = "xyes"])
+ 
+-AC_ARG_ENABLE(
+-  guile_1_8,
+-  AS_HELP_STRING([--enable-guile_1_8], [use GUILE_1_8 @<:@default=no@:>@]),
+-  [
+-    if test "x$enableval" != "xno"; then
+-      useguile_1_8=yes
+-      PKG_CHECK_MODULES(GUILE_1_8, guile-1.8 >= 1.8)
+-      LIBS="$LIBS $GUILE_1_8_LIBS"
+-      CFLAGS="$CFLAGS $GUILE_1_8_CFLAGS"
+-    fi
+-  ], [ useguile_1_8=no ])
+-
+-AC_ARG_ENABLE(
+-  guile_2_0,
+-  AS_HELP_STRING([--enable-guile_2_0], [use GUILE_2_0 @<:@default=no@:>@]),
+-  [
+-    if test "x$enableval" != "xno"; then
+-      useguile_2_0=yes
+-      PKG_CHECK_MODULES([GUILE_2], [guile-2.0])
+-      LIBS="$LIBS $GUILE_2_LIBS"
+-      CFLAGS="$CFLAGS $GUILE_2_CFLAGS"
+-    fi
+-  ], [ useguile_2_0=no ])
+-
+-if [test "x$useguile_1_8" = "xno"] && [test "x$useguile_2_0" = "xno"]; then
+-  PKG_CHECK_MODULES([GUILE_2], [guile-2.0], useguile_2_0=yes, useguile_2_0=no)
+-  LIBS="$LIBS $GUILE_2_LIBS"
+-  CFLAGS="$CFLAGS $GUILE_2_CFLAGS"
+-  if test "x$useguile_2_0" = "xno"; then
+-    PKG_CHECK_MODULES(GUILE_1_8, guile-1.8 >= 1.8)
+-    LIBS="$LIBS $GUILE_1_8_LIBS"
+-    CFLAGS="$CFLAGS $GUILE_1_8_CFLAGS"
+-    useguile_1_8=yes
+-  else
+-    useguile_2_0=yes
+-  fi
+-fi
++#AC_ARG_ENABLE(
```

Comment:
Please don't add commented out stuff in patch. This makes the patch much bigger than it needs to be, and has no value for users running the ebuild.

Reply:
That was the reason I asked for it above :) I'm going to remove it.